### ISSUE TITLE
qubes-doc appears to be unlicensed

### ISSUE THREAD

**Issue by strugee** [QubesOS/qubes-issues#4219]
Maybe I'm missing something super obvious, but the Qubes documentation appears to be unlicensed. https://github.com/QubesOS/qubes-doc doesn't mention it in the README or a LICENSE/COPYING file, and I don't see anything in https://www.qubes-os.org/doc/ either. The closest I can find is https://www.qubes-os.org/doc/license/, but that's not really right since that's about the code itself.

Thanks for maintaining Qubes <3

**Comment by andrewdavidwong** [comment]
CC: @rootkovska, @marmarek

**Comment by mfc** [comment]
shall we assume GPLv2 and add to qubes-doc repo? and if so broaden language on https://www.qubes-os.org/doc/license to explicitly include documentation?

**Comment by strugee** [comment]
I would personally feel comfortable licensing my work under GPLv2, but it doesn't seem like the best solution to me. GPL is meant for software; why not pick a license that's more suited for documentation like Creative Commons? You could also use the GNU Free Documentation License although honestly I don't know much about that one.

**Comment by andrewdavidwong** [comment]
How about CC BY-SA?

From https://creativecommons.org/licenses/:
> Attribution-ShareAlike 
> CC BY-SA
> 
> This license lets others remix, tweak, and build upon your work even for commercial purposes, as long as they credit you and license their new creations under the identical terms. This license is often compared to “copyleft” free and open source software licenses. All new works based on yours will carry the same license, so any derivatives will also allow commercial use. This is the license used by Wikipedia, and is recommended for materials that would benefit from incorporating content from Wikipedia and similarly licensed projects.

The [Fedora docs](https://docs.fedoraproject.org/en-US/docs/) also use this license (version 3.0).

What do you think @marmarek ?

**Comment by marmarek** [comment]
Yes, GPL is not good for documentation, but CC BY-SA seems like a good fit.

**Comment by strugee** [comment]
Don't we need to ask all copyright holders if they're okay with that license? Sorry for raining on the parade :/

**Comment by andrewdavidwong** [comment]
> Don't we need to ask all copyright holders if they're okay with that license? Sorry for raining on the parade :/

Do we? I have no idea. Who are all the copyright holders? Is it just everyone who has contributed to qubes-doc (hundreds of people)? How are we supposed to ask them all? What do we about the ones we can't reach or who don't respond?

If this is a legal requirement and we can't meet it (seems like we wouldn't be able to), then what are our alternatives? Can we just refuse to specify a license for the documentation?

**Comment by andrewdavidwong** [comment]
Reverted commits.

**Comment by andrewdavidwong** [comment]
Sounds like we need legal advice on this issue.

**Comment by marmarek** [comment]
cc @MiCh

**Comment by mfc** [comment]
FYI we're seeking expert feedback on this and will report back.

**Comment by andrewdavidwong** [comment]
> FYI we're seeking expert feedback on this and will report back.

Did anything ever come of this?

**Comment by mfc** [comment]
> > FYI we're seeking expert feedback on this and will report back.
> 
> Did anything ever come of this?

i re-upped email with current state

**Comment by mfc** [comment]
i'll loop you and the new documentation maintainers @maiska & @rapenne-s into a thread on the current state - your input would be very welcome

**Comment by mfc** [comment]
hi all, thank you for your patience. 

after consulting with lawyers at [Free Software Foundation Europe](https://en.wikipedia.org/wiki/Free_Software_Foundation_Europe) and internally with the team we have come to the conclusion that past documentation contributions were done under [GPLv2](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html). The [licensing information for the Qubes OS project](https://www.qubes-os.org/doc/license/) is very clear (especially since [2021](https://github.com/QubesOS/qubes-doc/pull/1145)) that it covers all content in the Qubes OS github project, which would include the `qubes-doc` repo.

GPLv2 is not considered a common license for documentation, so it is worth us reflecting on whether we want to have future contributions under a more common license for documentation, but those discussions will happen elsewhere than this issue (in the `qubes-project` mailing list, etc).

for making the existing licensing information more clear to contributors & users, we have the existing open issues:
- https://github.com/QubesOS/qubes-issues/issues/6500
- https://github.com/QubesOS/qubes-issues/issues/7320

i will close this issue as resolved in a couple weeks, to give folks time to provide any feedback.
